src/models/monster.js

```diff
@@ -4,6 +4,7 @@ var Schema = mongoose.Schema;
 var MonsterSchema = new Schema({
   index: String,
   name: String,
+  challenge_rating: [Number],
```

Comment:
Should this be a list? I assume `Schema` should match the shape of the actual model, but I could be wrong.

Reply:
It is a list, hence the `[]`. We split our query params, and then the `$in` takes care of it

Reply:
Ah! I think I don't quite understand how Schema works but thats on me.

Reply:
Hm, could be me too. Maybe it could be just a string? Not sure

Reply:
I'll try a few more ways

Reply:
Tried changing it to Number. Seems to still work... No idea what this is actually for, still learning lol